src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringsArr = sourceString.split(';');
+  const res = {};
+
+  stringsArr.forEach((str) => {
+    const clSrt = str.trim();
+
+    const ruleParam = clSrt.split(':');
+    if (ruleParam && ruleParam.length === 2) {
+      res[ruleParam[0].trim()] = ruleParam[1] ? ruleParam[1].trim() : '';
```

Comment:
The conditional `ruleParam[1] ? ruleParam[1].trim() : ''` ensures that if the value part is missing, an empty string is assigned. This is a good practice to handle potential missing values.